src/connector/Cargo.toml

```diff
@@ -46,6 +46,8 @@ clickhouse = { git = "https://github.com/risingwavelabs/clickhouse.rs", rev = "6
     "time",
 ] }
 csv = "1.3"
+datafusion = "33.0.0"
```

Comment:
Is the dependency `datafusion` used only in test?

Reply:
yes, query delta need to use it